api/V1/Recording.js

```diff
@@ -205,6 +205,7 @@ module.exports = (app, baseUrl) => {
         include: [
           { model: models.Group },
           { model: models.Tag },
+          { model: models.Device, where: {}, attributes: ["devicename", "id"] },
```

Comment:
Is the where necessary if it's empty?

Reply:
Without the `where: {}` i get an `SequelizeDatabaseError: column Recording.DeviceId does not exist` when doing an query. Not exactly sure why this is..